components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2AuthenticationConstant.java

```diff
@@ -24,10 +24,21 @@ private GoogleOAuth2AuthenticationConstant() {
     public static final String GOOGLE_AUTHZ_ENDPOINT = "GoogleAuthzEndpoint";
     public static final String GOOGLE_TOKEN_ENDPOINT = "GoogleTokenEndpoint";
     public static final String GOOGLE_USERINFO_ENDPOINT = "GoogleUserInfoEndpoint";
+    public static final String ENABLE_CSRF_VALIDATION_FOR_GOT = "EnableCSRFValidationForGOT";
     public static final String GOOGLE_CONNECTOR_FRIENDLY_NAME = "Google";
     public static final String GOOGLE_CONNECTOR_NAME = "GoogleOIDCAuthenticator";
     public static final String GOOGLE_SCOPE = "openid email profile";
     public static final String CALLBACK_URL = "Google-callback-url";
     public static final String ADDITIONAL_QUERY_PARAMS = "AdditionalQueryParameters";
     public static final String CLAIM_DIALECT_URI_PARAMETER = "ClaimDialectUri";
+
+    // Google One Tap property name.
+    public static final String GOOGLE_ONE_TAP_ENABLED = "IsGoogleOneTapEnabled";
+
+    // Google One Tap checkbox label in carbon console.
+    public static final String GOOGLE_ONE_TAP_DISPLAY_NAME = "Enable One Tap";
```

Comment:
Are we okay with `Enable One Tap` as the display name? Shouldn't it be `Google One Tap` if it is the display name?

Reply:
This name was discussed and approved at the demonstrations